### ISSUE TITLE
Upgrade hangs on 'Waiting for actor ID'

### ISSUE THREAD

**Issue by seedrick** [siderolabs/talos#7170]
I'm trying to install the proprietary nvidia drivers. I was able to build the image just fine, but when I launch `talosctl upgrade` all I ever get is `waiting for actor ID` in response. Note that when I tried `bootstrap` I got `rpc error: code = AlreadyExists desc = etcd data directory is not empty` as a reply, making me assume I must have fat-fingered it twice if that matters.

- Talos version: [1.4.1
- Kubernetes version: 1.26.1
- Platform: bare metal

**Comment by frezbo** [comment]
`waiting for actor ID` might mean it never pulled the installer image.

What does `talosctl dmesg` show after issuing the upgrade command?

**Comment by seedrick** [comment]
thanks for a lightning quick reply! Yes, looks like I messed up the link. I was trying to use a local registry initially but ran into issues so I just opted to toss everything on dockerhub. since the `push` didn't have an FQDN and such I didn't think I would need one, so the upgrade command just has `seedrick/installler:v1.4.0`.  If [this](https://hub.docker.com/layers/seedrick/installer/v1.4.0-nvidia/images/sha256-fae9706ddbaa6dae9696434cd030f0035f3c6e7f747f16e5b4a7ab33d08c58b0?context=repo) is the image I want to use, what's the proper form the URL should take? The usual suspects didn't work.
